addons/sale_stock/models/sale_order.py

```diff
@@ -215,13 +215,16 @@ class SaleOrderLine(models.Model):
     is_mto = fields.Boolean(compute='_compute_is_mto')
     display_qty_widget = fields.Boolean(compute='_compute_qty_to_deliver')
 
-    @api.depends('product_id', 'product_uom_qty', 'qty_delivered', 'state', 'product_uom')
+    @api.depends('product_type', 'product_uom_qty', 'qty_delivered', 'state', 'move_ids', 'product_uom')
```

Comment:
As we saw recently at least in v13 related fields don't behave well in computes, maybe we could leave it as it was or draw the full relation

Reply:
But this doesn't happen on `depends`. Take into account that this is working in 14.0, that uses the same ORM.

Reply:
Ok, then